erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -851,7 +851,33 @@ def set_work_order_ops(name):
 
 
 @frappe.whitelist()
-def make_stock_entry(work_order_id, purpose, qty=None, scrap_remaining=False, args=None):
+def finish_multiple_work_orders(work_orders, args=None):
+	if work_orders and isinstance(work_orders, str):
+		work_orders = json.loads(work_orders)
+
+	if not work_orders:
+		frappe.throw(_("Work Orders not selected"))
+
+	frappe.enqueue(make_stock_entry_against_multiple_work_orders, work_orders=work_orders, args=args)
```

Comment:
Need to call `make_stock_entry_against_multiple_work_orders.catch` and better to add `timeout=600`